build/dev-docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.12
+
+COPY dep-*.sha256 .
+
+RUN apt-get update && \
+    mkdir release && \
+    wget -q -O release/dep-$(go env GOOS)-$(go env GOARCH) https://github.com/golang/dep/releases/download/v0.5.0/dep-$(go env GOOS)-$(go env GOARCH) && \
+    sha256sum -c dep-$(go env GOOS)-$(go env GOARCH).sha256 && \
+    mv release/dep-$(go env GOOS)-$(go env GOARCH) /go/bin/dep && \
+    chmod +x /go/bin/dep && \
+    rmdir release && \
+    rm dep-*.sha256 && \
+    apt-get install -y protobuf-compiler && \
+    go get -u github.com/golang/protobuf/protoc-gen-go && \
+    mkdir -p /go/src/github.com/google && \
+    git clone --branch master https://github.com/google/protobuf /go/src/github.com/google/protobuf && \
+    go get -u golang.org/x/lint/golint && \
+    mkdir -p /go/src/github.com/onosproject/simulators
+
+WORKDIR "/go/src/github.com/onosproject/simulators"
```

Comment:
You can probably do this for now, but we should not push this to Docker hub.  Instead, we should figure out how to make this parametrizable via args. There is no reason to have nearly identical builder Docker images. This is the work that Jordan was alluding to earlier. We should probably just do this now/very soon.

Reply:
@tomikazi 
Yes. I was discussing it with Ray as well. We should have one image for build and send the working dir as an argument to the docker.